frontend/src/Frontend/UI/Dialogs/Signing.hs

```diff
@@ -70,6 +70,7 @@ uiSigning appCfg ideL signingRequest onCloseExternal = do
           , _deploymentSettingsConfig_gasLimit = _signingRequest_gasLimit signingRequest
           , _deploymentSettingsConfig_caps = Just $ _signingRequest_caps signingRequest
           , _deploymentSettingsConfig_extraSigners = fromPactPublicKey <$> fromMaybe [] (_signingRequest_extraSigners signingRequest)
+          , _deploymentSettingsConfig_includePreviewTab = False
```

Comment:
Why?

Reply:
Why a default or why this default?

Reply:
Signing requests should have preview screens - unless I've missed communication from Kadena

Reply:
Not sure - @mankyKitty ?

Reply:
Signing requests should show the preview screen.

Reply:
It does now